code/game/objects/items/cards_ids.dm

```diff
@@ -281,6 +281,8 @@
 	if(mining_points)
 		. += "There's [mining_points] mining equipment redemption point\s loaded onto this card."
 	. = ..()
+	if(istype(src, /obj/item/card/id/paper))  // forces off bank info for paper slip
+		return
```

Comment:
Should be a variable on the ID basetype saying if bank accounts are restricted rather than checking type (See all other instances of specifically checking for type).

Reply:
I am going to refactor this too once the paper slip is merged. I think you can try to merge and then see what I would do.

Reply:
so should i adjust it or wait until its merged and dragon fixes it

Reply:
let's see what Bacon would say

Reply:
I think it should just be a var on the ID